spec/features/world_index_spec.rb

```diff
@@ -1,76 +1,157 @@
 require "integration_spec_helper"
 
 RSpec.feature "World index page" do
-  before do
-    stub_content_store_has_item("/world", GovukSchemas::Example.find("world_index", example_name: "world_index"))
-    visit "/world"
-  end
+  context "without the GraphQL feature flag" do
```

Comment:
How gross would it be to do this iteratively? 

```
graphql_setup = {
   "on" = "with the GraphQL feature flag", 
   "off" = "without the GraphQL feature flag"
}

graphql_setup.each do |state, message|
 blah
  blah
   blah
```

Reply:
I've moved the tests into a `shared_example` then used `it_behaves_like` in each of the contexts, which has removed all the duplication.

Reply:
I don't have a chef kiss emoji on github 👨‍🍳 💋